reports/_setup.qmd

```diff
@@ -4,13 +4,15 @@ params:
   year: "2023"
 ---
 
-```{r}
+```{r setup_file}
```

Comment:
**issue:** This chunk actually can't be named because it runs multiple times in the full `performance.qmd` document (Quarto complains if chunks share the same name).